README.md

```diff
@@ -52,26 +52,33 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ![CoatRack architecture overview](./spring-boot/admin/src/main/resources/static/images/coatrack-architecture-overview.png)
 
-## Install
+## Installation for local usage
 
-Scripts for building and running CoatRack are available in the root folder of the code base. To build CoatRack, use:
+Go to the docker-compose-setup folder and execute:
 
-```console
-./build.sh
-```
-To run the CoatRack web application, use:
+ ```sh
+ .init.sh
+ ```
 
-```console
-./run-admin-application.sh
+The script only needs to be executed once at the very first time to initialize the docker volumes. After that, executing 'docker-compose up/down' is sufficient. After starting up, the CoatRack web application will be accessible at `http://localhost:8080`.
+
+If you want to reset CoatRack, stop/remove all containers, remove all volumes and start the ```init.sh``` script again.
```

Comment:
Just an idea: does it maybe make sense to create a "un-install" script for this?

Reply:
I think it is a nice and easy to implement feature which is why I just added it.